event-log/src/main/scala/io/renku/eventlog/events/consumers/projectsync/SubscriptionFactory.scala

```diff
@@ -23,25 +23,25 @@ import cats.syntax.all._
 import cats.{NonEmptyParallel, Parallel}
 import io.renku.eventlog.EventLogDB.SessionResource
 import io.renku.eventlog.Microservice
+import io.renku.eventlog.metrics.QueriesExecutionTimes
 import io.renku.events.consumers.EventHandler
 import io.renku.events.consumers.subscriptions.SubscriptionMechanism
 import io.renku.events.consumers.subscriptions.SubscriptionPayloadComposer.categoryAndUrlPayloadsComposerFactory
 import io.renku.graph.tokenrepository.AccessTokenFinder
 import io.renku.http.client.GitLabClient
-import io.renku.metrics.{LabeledHistogram, MetricsRegistry}
+import io.renku.metrics.MetricsRegistry
 import org.typelevel.log4cats.Logger
 
 object SubscriptionFactory {
 
   def apply[F[
       _
-  ]: Async: NonEmptyParallel: Parallel: GitLabClient: AccessTokenFinder: MetricsRegistry: SessionResource: Logger](
-      queriesExecTimes: LabeledHistogram[F]
-  ): F[(EventHandler[F], SubscriptionMechanism[F])] = for {
+  ]: Async: NonEmptyParallel: Parallel: GitLabClient: AccessTokenFinder: SessionResource: Logger: MetricsRegistry: QueriesExecutionTimes]
```

Comment:
I think Parallel extends NonEmptyParallel, so we could omit NonEmptyParallel from that list (but yeah, not much to gain :))